.github/dependabot.yml

```
@@ -15,6 +15,7 @@ updates:
       - dependency-name: "@automattic/*"
       - dependency-name: "@electron-forge/*"
       - dependency-name: "@wordpress/*"
+      - dependency-name: "@wp-playground/*"
```

Let's also add `@php-wasm/*`

```suggestion
      - dependency-name: "@wp-playground/*"
      - dependency-name: "@php-wasm/*"
```

---

Thanks, added.